cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -316,8 +324,8 @@ Add Service Wizard: Customize Role Assignments
    :align: center
    :class: bordered-image
 
-   **Add Service Wizard, Page 3:** Completing assignments with CDAP Gateway client
-   added to all nodes of the cluster including those with CDAP roles.
+   **Add Service Wizard, Page 2 (dialog):** Completing assignments with the CDAP Gateway client
+   added to other nodes of the cluster; it can be added to those with CDAP roles.
```

Comment:
" it can also be added to nodes with CDAP roles." might be a little less confusing?